compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangPackageBuilder.java

```diff
@@ -1491,7 +1491,7 @@ void endFunctionDef(DiagnosticPos pos, Set<Whitespace> ws, boolean publicFunc, b
     }
 
     void startWorker(PackageID pkgID) {
-        this.startLambdaFunctionDef(pkgID);
+        this.startLambdaFunctionDef(pkgID, 0);
```

Comment:
Shall we take 0 to a meaningful constant??

Reply:
Or even better to get rid of `annotCount` from `startLambdaFunctionDef` because Lambdas can't have annotations anyway.

Reply:
Resolved